src/main/resources/application.properties.j2

```diff
@@ -37,6 +37,9 @@ allow_graphql_mutation=false
 # redis.host =
 # redis.port =
 redis.ttl=-1
+# redis.authentication.enabled=false
```

Comment:
Since all server environments have authentication enabled, I think this line should be uncommented and set to true.

Reply:
Updated